LICENSE

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2015 Gigwalk
```

Comment:
Yeah oops, didn't meant to have that change, I left him as the author in the package.json